{"name": "Create order with ingredients", "status": "broken", "statusDetails": {"message": "NameError: name 'Urls' is not defined", "trace": "self = <test_create_order.TestCreateOrder object at 0x000001EC62F34510>, api_client = <helpers.api_client.APIClient object at 0x000001EC62EF7B10>\nregistered_user = {'email': 'anitarobinson@example.com', 'password': 'Nx0Nca5_#c', 'token': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...WNlOWVkMjgwMDAxYjY3ZjA1OCIsImlhdCI6MTc1NTQyOTU4MiwiZXhwIjoxNzU1NDMwNzgyfQ.NK70ARWxjJC3yyZ2B5YeoRe8sjjCWLceDy3F9ze525U'}\n\n    @allure.title('Create order with ingredients')\n    def test_create_order_with_ingredients(self, api_client, registered_user):\n>       ingredients = get_valid_ingredients(api_client)\n\ntests\\test_create_order.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\napi_client = <helpers.api_client.APIClient object at 0x000001EC62EF7B10>\n\n    def get_valid_ingredients(api_client):\n>       response = api_client.get(Urls.INGREDIENTS)\nE       NameError: name 'Urls' is not defined\n\nhelpers\\data_generator.py:16: NameError"}, "start": 1755429581281, "stop": 1755429581281, "uuid": "26b03ff7-d7dd-4d80-9b45-a48ecaeca771", "historyId": "7c86c6e6b94b01b7af6e86a42ca72c43", "testCaseId": "7c86c6e6b94b01b7af6e86a42ca72c43", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_ingredients", "labels": [{"name": "feature", "value": "Order Creation"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "dea-1611"}, {"name": "thread", "value": "12860-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}